projects/plugins/jetpack/modules/sso/class.jetpack-sso-user-admin.php

```diff
@@ -26,7 +26,6 @@ class Jetpack_SSO_User_Admin {
 		 * Constructor function.
 		 */
 		public function __construct() {
-			add_action( 'delete_user', array( 'Jetpack_SSO_Helpers', 'delete_connection_for_user' ) );
```

Comment:
not being used action

Reply:
Can you elaborate on this a change a bit more?
Is it not needed ?

Reply:
It doesn't look necessary, the `delete_connection_for_user` function does not even exist in this file.

Reply:
The function is called from the Jetpack_SSO_helpers class in `class.jetpack-sso-helpers.php`. 

We keep this for now and investigate/remove if necessary in a separate PR.

Reply:
We can tackle it in another PR and keep this one with one focus only 👍 added it back.

Reply:
Yes! But I mean, we cannot delete this. ` array( 'Jetpack_SSO_Helpers',` means that you have to search for that function in the `Jetpack_SSO_Helpers` file. It's like when we import utils in React. And it is necessary as to delete the SSO connection to the site when we delete an user!

Reply:
Also @renancarvalho, remember to put this back, as in the code changes I still see it being removed.

Reply:
Sure 👍

Reply:
This is something that was already there even before the sso changes
https://github.com/Automattic/jetpack/pull/35617/files#diff-843ba2ff1530e02eeb7d6c18570ad09c89a3a1ad5aacebd7affee00e6ddb616fL49
It removes the jetpack connection when deleting a user if he is connected